processor/src/decoder/tests.rs

```diff
@@ -0,0 +1,675 @@
+use super::{super::DecoderTrace, Felt, Operation, Word, NUM_OP_BITS};
+use crate::{ExecutionTrace, Process, ProgramInputs};
+use vm_core::{
+    decoder::{
+        ADDR_COL_IDX, GROUP_COUNT_COL_IDX, HASHER_STATE_RANGE, IN_SPAN_COL_IDX, OP_BITS_OFFSET,
+        OP_BITS_RANGE, OP_INDEX_COL_IDX,
+    },
+    program::blocks::{CodeBlock, Span},
+    FieldElement, StarkField, DECODER_TRACE_RANGE,
+};
+
+// CONSTANTS
+// ================================================================================================
+
+const ONE: Felt = Felt::ONE;
+const ZERO: Felt = Felt::ZERO;
```

Comment:
I like the readability of just using ONE and ZERO, and I notice you've done this in 3 or 4 files. What would you think of putting this in core and doing it everywhere?

Reply:
Yes - good idea! But I'll leave it for future PRs.